.travis.yml

```diff
@@ -11,4 +11,5 @@ script:
   - docker --help
   - dockerd --help
   - docker pull hello-world #for some IT cases
+  - docker run -d --privileged -p 12375:2375 docker:dind
```

Comment:
@sbespalov Can you shortly explain what this line does? 

I assume it will start a docker instance which will listen at some port, locally. But will you be able to connect to it via TCP? That's what we need, not the default Unix Socket connection.

Reply:
> I assume it will start a docker instance which will listen at some port, locally
yes. It was done according [this](https://github.com/amihaiemil/docker-java-api/issues/178#issuecomment-444029856) comment.

>But will you be able to connect to it via TCP? 
seems it is available by TCP

```
$ sudo docker run -d --privileged -p 12375:2375 docker:dind
86189e1b63b0a87fbb83ca543830bdae9c5ef71ff066f47ec7bb39387cec713e
$ sudo docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                     NAMES
86189e1b63b0        docker:dind         "dockerd-entrypoint.…"   7 seconds ago       Up 4 seconds        0.0.0.0:12375->2375/tcp   compassionate_noether
$ telnet localhost 12375
Trying 127.0.0.1...
Connected to localhost.
Escape character is '^]'.
]
HTTP/1.1 400 Bad Request
Content-Type: text/plain; charset=utf-8
Connection: close

400 Bad RequestConnection closed by foreign host.
```

Reply:
@sbespalov Can you please also give the container a name and stop it after the tests?

Reply:
@bkuzmic We don't need to start any container, only to have a Docker running, that we can connect to from the tests and play around :)

Reply:
@amihaiemil I don't know what you mean. The "docker run" command starts the container. Which means that we need to stop it too after the mvn command.